libraries/AP_GyroFFT/AP_GyroFFT.cpp

```diff
@@ -53,15 +67,15 @@ const AP_Param::GroupInfo AP_GyroFFT::var_info[] = {
     // @Param: MINHZ
     // @DisplayName: Minimum Frequency
     // @Description: Lower bound of FFT frequency detection in Hz.
-    // @Range: 10 400
+    // @Range: 80 400
```

Comment:
why 80? That seems pretty high. I've setup notch filters down to 20Hz on some vehicles (eg. helicopters). Does it really not work at lower frequencies?

Reply:
It's not that it doesn't work, it's that there is an increasing chance of it picking up the big spike at very low frequencies (due to control input etc). I wanted a default range that was known to be safe.

Reply:
this isn't a default range, it is the limit that the user can set with the GCS UI (some GCS allow you to go past this if you force it)